src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -7750,5 +7463,6851 @@ void M2()
+                );
+
+            ValidatePropertyModifiers_10(compilation1);
+        }
+
+        private static void ValidatePropertyModifiers_10(CSharpCompilation compilation1)
+        {
+            var test1 = compilation1.GetTypeByMetadataName("Test1");
+            var i1 = compilation1.GetTypeByMetadataName("I1");
+            var p1 = i1.GetMembers().OfType<PropertySymbol>().ElementAt(0);
+            var p1get = p1.GetMethod;
+
+            Assert.True(p1.IsAbstract);
+            Assert.False(p1.IsVirtual);
+            Assert.False(p1.IsSealed);
+            Assert.False(p1.IsStatic);
+            Assert.False(p1.IsExtern);
+            Assert.False(p1.IsOverride);
+            Assert.Equal(Accessibility.Private, p1.DeclaredAccessibility);
+            Assert.Null(test1.FindImplementationForInterfaceMember(p1));
+
+            Assert.True(p1get.IsAbstract);
+            Assert.False(p1get.IsVirtual);
+            Assert.True(p1get.IsMetadataVirtual());
+            Assert.False(p1get.IsSealed);
+            Assert.False(p1get.IsStatic);
+            Assert.False(p1get.IsExtern);
+            Assert.False(p1get.IsAsync);
+            Assert.False(p1get.IsOverride);
+            Assert.Equal(Accessibility.Private, p1get.DeclaredAccessibility);
+            Assert.Null(test1.FindImplementationForInterfaceMember(p1get));
+
+            var p2 = i1.GetMembers().OfType<PropertySymbol>().ElementAt(1);
+            var p2get = p2.GetMethod;
+
+            Assert.False(p2.IsAbstract);
+            Assert.True(p2.IsVirtual);
+            Assert.False(p2.IsSealed);
+            Assert.False(p2.IsStatic);
+            Assert.False(p2.IsExtern);
+            Assert.False(p2.IsOverride);
+            Assert.Equal(Accessibility.Private, p2.DeclaredAccessibility);
+            Assert.Same(p2, test1.FindImplementationForInterfaceMember(p2));
+
+            Assert.False(p2get.IsAbstract);
+            Assert.True(p2get.IsVirtual);
+            Assert.True(p2get.IsMetadataVirtual());
+            Assert.False(p2get.IsSealed);
+            Assert.False(p2get.IsStatic);
+            Assert.False(p2get.IsExtern);
+            Assert.False(p2get.IsAsync);
+            Assert.False(p2get.IsOverride);
+            Assert.Equal(Accessibility.Private, p2get.DeclaredAccessibility);
+            Assert.Same(p2get, test1.FindImplementationForInterfaceMember(p2get));
+
+            var p3 = i1.GetMembers().OfType<PropertySymbol>().ElementAt(2);
+
+            Assert.False(p3.IsAbstract);
+            Assert.False(p3.IsVirtual);
+            Assert.True(p3.IsSealed);
+            Assert.False(p3.IsStatic);
+            Assert.False(p3.IsExtern);
+            Assert.False(p3.IsOverride);
+            Assert.Equal(Accessibility.Private, p3.DeclaredAccessibility);
+            Assert.Null(test1.FindImplementationForInterfaceMember(p3));
+
+            ValidateP3Accessor(p3.GetMetho
```

Comment:
Agreed, this does look wrong. I don't know what you mean saying it is an old bug; we didn't have private interface members except as "bad" metadata before. Do you have an issue tracking this as a bug in the implementation (it is more than merely a bug in the test, which is what a comment appearing here in the tests would suggest).